better-code/src/chapter-2-contracts.md

```diff
@@ -926,14 +929,16 @@ the elements arranged from least to greatest. The contract gives an
 explicit precondition that isn't implied by the summary: it requires
 that the predicate be a strict weak ordering.
 
+<!-- SRP: this section bothers me. "among others" instead of fully spelling out the requirements, using (i, j + 1) which may not exist, and the n^2 comparisons without calling out the O(n^3) complexity or which properties could be practically checked. Also is "stable" the term we want to use? Regular and deterministic are also candidates. I've tried to rewrite this a couple of times, but it just gets too complex and the main point is lost. -->
```

Comment:
I think you may have some valid points here but too many complaints and a little to diffuse to deal with here; let's chat in person.

Reply:
I do think "deterministic" is a better word than "stable".